pkg/gitprovider/types.go

```diff
@@ -3,11 +3,21 @@
 
 package gitprovider
 
+type SigningMethod string // @name SigningMethod
+
+const (
+	SigningMethodSSH   SigningMethod = "ssh"
+	SigningMethodGPG   SigningMethod = "gpg"
+	siginingMethodNone SigningMethod = "none"
```

Comment:
@bryans-go I had something else in mind with https://github.com/daytonaio/daytona/pull/1146/files#r1778545525.

`"none"` shouldn't be defined as an option here.

The solution I was aiming for included setting up a variable inside `GitProviderSelectionView` that would be a helper variable for selecting the signing method. It would include `ssh` `gpg` and `none` as options. After the form runs inside that function, you would check the value of that variable and assign `GitProviderView`  to the appropriate value (if `none` then `nil`).

Reply:
To clarify, this approach scopes the choice to that method which is view-only. In that case, the domain part of the app does not deal with the view which should be the preferred approach.

Reply:
ok.